strapi/strapi
# feat(history): restore version

### What does it do?

- Adds the logic to a version 
- Adds a simple dumb button on the FE + sets up the RTK mutation to make it easier to test

### Why is it needed?

- To restore a history version

### How to test it?

- Create a history version for an entry
- Create another with some changes
- Go to history and restore the previous version
- It should create a new history version that restore your selected version


Test relations and media
- Add relations/media to an entry, create a history version
- Go delete the relation/media entries/assets, create another history version for the entry
- Restore the version that had the relations/media
- The deleted relations should not be restored, existing relations should be restored

Unknown fields:
- Added fields should be in the restored version but with an empty value
- Deleted fields should not be restored

Not supported:
- polymorphic relations / components

NOTE

To ensure this work makes it into the beta blitz on monday tests will be handled in another PR CS-752
